client/termdb/TermTypeSearch.ts

```diff
@@ -14,7 +14,7 @@ The tree target is used to determine the allowed term types.
 
 const useCasesExcluded = {
 	matrix: [TermTypeGroups.SNP_LOCUS, TermTypeGroups.SNP_LIST],
-	filter: [TermTypeGroups.SNP_LOCUS, TermTypeGroups.SNP_LIST],
+	filter: [TermTypeGroups.SNP_LOCUS, TermTypeGroups.SNP_LIST, TermTypeGroups.GENE_EXPRESSION],
```

Comment:
@xzhou82 not sure if this is sufficient, or there has to be extra logic/detail required to know that geneExpression tab/term.type is not supposed to be available as a filter option.

Reply:
shouldn't do it. only gdc doesn't support gene exp filtering. non-gdc ds all support it. see get_geneExpression() of termdb.filter.js

as we discussed before, it needs to allow ds-specific override